packages/framework-core/src/services/graphql/graphql-query-generator.ts

```diff
@@ -78,6 +79,26 @@ export class GraphQLQueryGenerator {
     return queries
   }
 
+  private generateFilterListedQueries(): GraphQLFieldConfigMap<unknown, GraphQLResolverContext> {
+    const queries: GraphQLFieldConfigMap<unknown, GraphQLResolverContext> = {}
+    for (const name in this.targetTypes) {
+      const type = this.targetTypes[name]
+      const graphQLType = this.typeInformer.getGraphQLTypeFor(type.class)
+      queries[`List${inflected.pluralize(name)}`] = {
+        type: new GraphQLObjectType({
+          name: `${type.class.name}Connection`,
+          fields: {
+            items: { type: new GraphQLList(graphQLType) },
+            cursor: { type: GraphQLJSONObject },
+          },
```

Comment:
Is the recursive reference to `graphQLType` working even though `fields` is not a function?